src/main/scala/ru/d4team/chat/models/message/MessageRecord.scala

```diff
@@ -0,0 +1,18 @@
+package ru.d4team.chat.models.message
+
+import io.scalaland.chimney.dsl.TransformerOps
+
+import java.time.Instant
+import java.util.UUID
+
+final case class MessageRecord(
+    id: UUID,
+    chatRoomId: UUID,
+    writerId: UUID,
+    createdAt: Instant,
+    content: String
```

Comment:
With reference to the `chat_room_messanging` table, fields have to be named in the same way as in DB, but in lower camel case style:

- messageId
- chatId
- writerId
- message
- createdAt

Otherwise you will face with errors once start performing queries.

Reply:
corrected